checks/check_extra95

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# Prowler - the handy cloud security tool (copyright 2018) by Toni de la Fuente
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+CHECK_ID_extra95="9.5"
+CHECK_TITLE_extra95="[extra95] Check if there is any unassigned elastic ip's (Not Scored) (Not part of CIS benchmark) (Custom Check)"
+CHECK_SCORED_extra95="NOT_SCORED"
+CHECK_TYPE_extra95="EXTRA"
+CHECK_SEVERITY_extra95="Critical"
```

Comment:
I don't think is actually `Critical`, since it's only showing if it's generating an extra cost or not

Reply:
Agree, It can be consider as Low or Medium